lib/locales/ja.js

```diff
@@ -11,7 +11,64 @@
+        'Pan': '拡大',                                                                                     // components/modebar/buttons.js:94
+        'Produced with Plotly': 'Plotlyにより作成',                                                        // components/modebar/modebar.js:256
+        'Reset': 'リセット',                                                                                // components/modebar/buttons.js:432
+        'Reset axes': '軸をリセット',                                                                       // components/modebar/buttons.js:148
+        'Reset camera to default': 'カメラをデフォルトに戻す',                                              // components/modebar/buttons.js:314
+        'Reset camera to last save': 'カメラを最後の保存状態に戻す',                                        // components/modebar/buttons.js:322
+        'Reset view': 'ビューをリセット',                                                                   // components/modebar/buttons.js:583
+        'Reset views': 'ビューをリセット',                                                                  // components/modebar/buttons.js:529
+        'Show closest data on hover': 'ホバー時に一番近いデータを表示',                                     // components/modebar/buttons.js:157
+        'Snapshot succeeded': 'スナップショットに成功',                                                     // components/modebar/buttons.js:66
+        'Sorry, there was a problem downloading your snapshot!': 'すみません、スナップショットダウンロードでエラーです!', // components/modebar/buttons.js:69
+        'Taking snapshot - this may take a few seconds': 'スナップショットを撮影',                          // components/modebar/buttons.js:57
+        'Zoom': 'ズーム',                                                                                   // components/modebar/buttons.js:85
+        'Zoom in': '拡大',                                                                                  // components/modebar/buttons.js:121
+        'Zoom out': '縮小',                                                                                 // components/modebar/buttons.js:130
+        'close:': '閉じる:',                                                                                // traces/ohlc/transform.js:139
+        'trace': 'トレース:',                                                                               // plots/plots.js:439
+        'lat:': '緯度:',                                                                                    // traces/scattergeo/calc.js:48
+        'lon:': '経度:',                                                                                    // traces/scattergeo/calc.js:49
+        'q1:': 'q1:',                                                                                       // traces/box/calc.js:130
+        'q3:': 'q3:',                                                                                       // traces/box/calc.js:131
```

Comment:
Are q1 and q3 OK left in English? These are the first and third quartiles, in a box plot.